bpf/lib/common.h

```diff
@@ -736,10 +739,10 @@ struct lb6_service {
 		__u32 affinity_timeout;	/* In seconds, only for svc frontend */
 	};
 	__u16 count;
+	__u16 local_count;
 	__u16 rev_nat_index;
 	__u8 flags;
 	__u8 flags2;
-	__u8 pad[2];
```

Comment:
This would unfortunately break connectivity on upgrades/downgrades of the agent given the structure change. If we need to extend, then we would need to use the pad area.